Vaadin/capstone-2024/src/main/java/com/example/application/views/collegebasketballprediction/CollegeBasketballPredictionsView.java

```diff
@@ -126,6 +127,19 @@ private void setComboBoxData(ComboBox comboBox) {
                         "Southern", "Southland", "Southwestern Athletic", "Summit League",
                         "Sun Belt", "West Coast", "Western Athletic"));
 
+         */
+
+        // List of conferences
+        List<String> items = new ArrayList<>();
```

Comment:
Why not just do something like:

final List<String> items = new ArrayList<>( conferenceNames );
items.add( 0, "ALL" );
comboBox.setItems( items );